scripts/scil_surface_create.py

```diff
@@ -0,0 +1,170 @@
+    morpho_g.add_argument('--erosion',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Erosion: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--dilation',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Dilation: number of iterations. [%(default)s]')
+    morpho_g.add_argument('--opening',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Opening (dilation of the erosion): number '
+                               'of iterations. [%(default)s]')
+    morpho_g.add_argument('--closing',
+                          type=ranged_type(int, 0, None, min_excluded=False),
+                          help='Closing (erosion of the dilation): number '
+                               'of iterations. [%(default)s]')
+
+    p.add_argument('--fill', action='store_true',
+                   help='Fill holes in the image.')
+
+    p.add_argument('--vox2vtk', action='store_true',
+                   help='Transformation to vox2vtk.')
+
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+
+    assert_inputs_exist(parser, [], [args.in_labels, args.in_mask])
+    assert_outputs_exist(parser, args, args.out_surface)
+
+    if args.in_labels:
+        # Load volume
+        labels_img = nib.load(args.in_labels)
+        labels_volume = get_data_as_labels(labels_img)
+
+        # Removed indices
+        if args.indices:
+            mask = merge_labels_into_mask(labels_volume,
+                                          ' '.join(args.indices))
+        else:
+            logging.warning('No indices provided, '
+                            'it will use all indices.')
+            mask = merge_labels_into_mask(labels_volume,
+                                          "1:" + str(labels_volume.max()))
+    else:
+        # Load mask
+        mask_img = nib.load(args.in_mask)
+        mask = get_data_as_mask(mask_img)
+
+    # Basic morphology
+    if args.erosion is not None:
+        mask = binary_erosion(mask, iterations=args.erosion)
+    if args.dilation is not None:
+        mask = binary_dilation(mask, iterations=args.dilation)
+    if args.opening is not None:
+        mask = binary_opening(mask, iterations=args.opening)
+    if args.closing is not None:
+        mask = binary_closing(mask, iterations=args.closing)
+
+    if args.fill:
+        mask = binary_fill_holes(mask)
+
+    # Extract marching cube surface from mask
+    vertices, triangles = mcubes.marching_cubes(mask, args.value)
+
+    # Generate mesh
+    mesh = trimeshpy.trimesh_vtk.TriMesh_Vtk(triangles.astype(int), vertices)
+
+    # Transformation based on the Nifti affine
+    if not args.vox2vtk:
```

Comment:
see my previous comment on "vox2vtk", which is now reversed since a "not" was added in front of the argument